docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -64,22 +70,37 @@ The output includes::
   User name for authentication with the server where to migrate data from
   ----
 
-These are the settings we'll use when configuring the migration in the next step.
-
-Perform the Migration
-'''''''''''''''''''''
-
-The following is the migration process:
-
-1. If you don't have a Aiven for PostgreSQL database already, run the following commands to create a couple of PostgreSQL services via :doc:`../../../tools/cli` substituting the parameters accordingly::
+  Variables
```

Comment:
There's something strange with the rendering here, try adding another blank line before the title, or removing any indentation.